rep-2000.rst

```diff
@@ -1304,6 +1201,66 @@ Build System Support:
++==========================+========================+===============+============================+================================+====================+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS      | Tier 1        | All Platforms              | All Architectures              | 3.2.2              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+--------------------+
+| rmw_connextdds           | RTI Connext            | Tier 1        | Ubuntu, Windows, and macOS | All Architectures except arm64 | 7.3.0.0            |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+--------------------+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS    | Tier 1        | All Platforms              | All Architectures              | 0.10.5             |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+--------------------+
+| rmw_zenoh_cpp            | Eclipse Zenoh          | Tier 1        | All Platforms              | All Architectures              | 1.4.0              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+--------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS      | Tier 2        | All Platforms              | All Architectures              | 3.2.2              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+--------------------+
+| rmw_gurumdds_cpp         | GurumNetworks GurumDDS | Tier 3        | Ubuntu and Windows         | All Architectures except arm32 | 4.2.0              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+--------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Platforms and Dependency Requirements:
+
++-----------------+---------------------------------------------------------+-------------------------------------------+----------------------------------------------------------------+
+|                 | Tier 1 (Full Support) [d][a][s]                         | Tier 2 (Testing and CI Support) [d][a][s] | Tier 3 (Community Support) [s]                                 |
++-----------------+------------------------------------+--------------------+-------------------------------------------+---------------+------------------------------+-----------------+
+| Package         | Ubuntu Noble (24.04) (amd64/arm64) | Windows 10 (amd64) | RHEL 9 (amd64)                            | macOS (arm64) | Debian Bookworm (12) (amd64) | OpenEmbedded    |
```

Comment:
Looking at this table, Windows 10 falls under `Tier 1 (Full Support) [d][a][s]` now, but it previously was just `Tier 1 [a][s]`, without the `[d]`. The way `[d]` and `[a]` are written, I don't think `[d]` applies to Windows 10: it has binaries available as a single archive (e.g., `ros2-kilted-*-windows-release-amd64.zip`), but no "Distribution-specific (Debian, RPM, etc.) packages."